docs/sources/review/run-a-local-webserver/index.md

```diff
@@ -72,6 +72,40 @@ The format is `<PROJECT>[:VERSION[:REPOSITORY[:DIR]]].`
 The example mounts the `PROJECT` tempo, at the default `VERSION` latest, using the `REPOSITORY` `tempo-doc-work`, and the default `DIR` `docs/sources`.
 This example builds the Tempo documentation from the local working directory, `tempo-doc-work`, instead of the standard `tempo` directory.
 
+## Understand Hugo output from `make docs`
+
+When you run `make docs`, Hugo (our static site generator) processes the Markdown files and outputs warnings and error messages.
+
+These messages are in the following format:
+
+```text
+WARN <DATE> <TIME> <LANGUAGE> REF_NOT_FOUND: Ref <RELREF ARGUMENT>: “<SOURCE FILE>:<LINE>:<COLUMN>”: <ERROR>
+```
+
+where:
+
+- `SOURCE FILE` is the file with the broken `relref`
+- `RELREF ARGUMENT` is the argument to the `relref` shortcode that is not working.
+- ` ERROR`` is the reason  `RELREF ARGUMENT` is not working.
+
+When you save a file with an active local build, the page is rechecked. If the error messages is not repeated, then the issue is fixed.
+
+### Example: Page not found
+
+In this example, Hugo is reporting a `page not found` error for a link to `alerting/set-up/migrating-alerts/opt-out` in the file `access-ontro/rbac-fixed-basic-role-definitions/index.md` on line 100.
+
+For more information about linking, refer to [Links and cross references]({{< relref "../../write/references" >}}).
+
+![Hugo output for running make docs](/media/docs/writers-toolkit/screenshot-make-docs-output.png)
+
+### Example: Rebuild failed due to shortcode
+
+In this example, the rebuild fails becuase the file `contribute-documentation/_index.md` is missing a closing shortcode for `{{% /admonition %}}` on line 152.
```

Comment:
```suggestion
In this example, the rebuild fails because the file `contribute-documentation/_index.md` is missing a closing shortcode for `{{% /admonition %}}` on line 152.
```